android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetActivity.java

```diff
@@ -37,6 +38,28 @@
 
     protected static final String TAG = JitsiMeetActivity.class.getSimpleName();
 
+    public static final String ACTION_JITSI_MEET_CONFERENCE = "org.jitsi.meet.CONFERENCE";
+    public static final String JITSI_MEET_CONFERENCE_OPTIONS = "JitsiMeetConferenceOptions";
+
+    // Helpers for starting the activity
+    //
+
+    public static void launch(Context context, JitsiMeetConferenceOptions options) {
+        // In case someone subclasses JitsiMeetActivity
+        Class currentClass = new Object() { }.getClass().getEnclosingClass();
```

Comment:
I'm not sure if I'm a big fan of this trick. I would probably prefer a template method which takes a class as the parameter. That would keep the class under control.

Reply:
```
public static <T extends JitsiMeetActivity> void launch(Context context, JitsiMeetConferenceOptions options, T activityType) {
        // In case someone subclasses JitsiMeetActivity
        Class activityClass = activityType.getClass();

        Intent intent = new Intent(context, activityClass);
        intent.setAction(ACTION_JITSI_MEET_CONFERENCE);
        intent.putExtra(JITSI_MEET_CONFERENCE_OPTIONS, options);
        context.startActivity(intent);
    }
```

Reply:
I was just thinking that maybe the enclosing class would not work if you subclass twice, but actually it should work, so feel free to disregard my comment.

Reply:
I like that suggestion! I only used the other trick because I didn't know better. Do I need to make both methods templated or can I leave the one taking String alone?